task/buildah-rhtap/0.1/buildah-rhtap.yaml

```diff
@@ -35,6 +35,8 @@ spec:
     name: IMAGE_URL
   - description: Digests of the base images used for build
     name: BASE_IMAGES_DIGESTS
+  - description: Link to the SBOM layer pushed to the registry as part of an OCI artifact.
+    name: SBOM_BLOB_URL
```

Comment:
If the buildah task has a README, it would be good to document how this url is meant to be used. How does one even download the SBOM from it and verify its  checksum?

Reply:
As well as document why it's the layer url and not the url of the whole artifact

Reply:
I still need to add a README to it. For reference, we can fetch the layer using:
```
oras blob fetch quay.io/bpimente/test-images@sha256:509e7e73b03bb9c02f6e14a80acc81296396f538834dff09013f134f35a98591 --output -
```

Is it ok if the README is added as a follow up PR?